docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
-
-Each general has lieutenants responsible for sending and receiving messages. However, Lieutenants may send traitor messages from a traitor general leading one general receiving the message to retreat. 
-
-If however, there are 3 Generals, but 2 Generals have **m** traitor Lieutenants passing traitor messages to those generals. This means that 1 General has **(m-1)** honourable Lieutenants passing accurate messages to that General.
-
-In short, Lamport, Shostak, and Pease found that no solution with fewer than 3m + 1 generals can cope with **m** traitors. In other words, consensus is impossible to achieve if a third or more of the generals are traitors.
+## The key aim of Cudos network tokenomics is :
 
+***To ensure that network participants are incentivised to work towards the greater good of the network. Specifically,ensuring the security, integrity and longevity of the network.***
 :::
 
-Use of Tendermint enables consensus to be achieved on the state of the blockchain even if up to ⅓ (~33%) of network machines are bad actors or fail in arbitrary ways.
-
-Second, incentivisation of Validator operators to cooperate behaviour on the Cudos network is built into the tokenomics model whereby those who secure the network by running a Validator are required to stake or self-delegate a minimum of 2M CUDOS in return for network rewards. Staking or self-delegating higher amounts increases the chances of that Validator being selected to conduct block validation as well as increasing their rewards as a proportion of the overall amount staked on the network. 
-
-Validators and Delegators earn block rewards for keeping the blockchain operational and secure.
-Block rewards consist of Staking rewards and Transaction fees.
-
-## Staking
-
-The act of staking tokens is called delegating.
-
-A validator needs to self-delegate at least 2M CUDOS.
+Tokenomics draws from game theory models that assume individual rational network participants are always working to maximise their own utility. ***The aim of tokenomics is to ensure that individual utility is best maximised by advancing overall network utility***. 
 
-A delegator is someone who delegates any amount of tokens to one or more validators.
+From this perspective, the **Cudos Network** tackles this challenge in multiple ways. 
 
-Tokens delegated to a validator, including its self-delegated tokens, are taken out of the delegator’s account and kept in a pool.
+### Consensus layer
 
-The validator and its delegators share the rewards based on the amounts they have contributed to this pool.
+First, the Cudos network uses **Tendermint** to provide a **consensus layer** and agree the state of the Cudos chain. Tendermint is **Byzantine Fault Tolerance (BFT)** (solves the [Byzantine Generals Problem](https://lamport.azurewebsites.net/pubs/byz.pdf)) enabling consensus to be achieved on the state of the blockchain even if up to ⅓ (~33%) of network machines are bad actors or fail in arbitrary ways.
```

Comment:
I think it should either be something like "Tendermint in Byzantine Fault tolerant", or "Tendermint **employs** BFT"

Reply:
Changed to Byzantine Fault Tolerant. 👍🏽